src/pysoem/pysoem.pyx

```diff
@@ -854,6 +893,8 @@ cdef class CdefSlave:
         if self._ecx_contextt == NULL:
             raise UnboundLocalError()
```

Comment:
Same here: This two lines could actually be removed, right? As `__check_context_is_initialized` is doing the same?